README.md

```diff
@@ -18,6 +18,12 @@ Requirements
 The plugin needs a database. The latest version of the schema can be found [here](https://github.com/killbill/killbill-dwolla-plugin/blob/master/src/main/resources/ddl.sql).
 
 Merchants need to create a Dwolla [application](https://developers.dwolla.com/guides/sandbox-setup/02-create-application.html).
+The first time a token pair needs to be created and saves into `dwolla_tokens` table.
```

Comment:
Could we add a private endpoint for this (in the servlet)? It would it more convenient.